src/ansys/fluent/visualization/contour.py

```diff
@@ -1,8 +1,24 @@
+"""Contour objects based on field name and surfaces list."""
+
 from typing import List, Optional
 
 
 class Contour:
+    """Provides contour objects based on field name and surfaces list.
+
+    Parameters
+    ----------
+    field : str
+        field name
+
+    surfaces : List[str]
+        surfaces list
+    """
+
     def _error_check(self, solver):
+        """
+        Checks field and surface names.
```

Comment:
```suggestion
        Check field and surface names.
```
Method docstring begins with a verb without an "s" or "es". Class descriptions, on the other hand, begin with verbs ending with "s" or "es".

Reply:
Done. Thank you.